assignment.js

```diff
@@ -33,6 +34,15 @@ console.log('Problem 1:')
 
 // Add your code below this line
 
+
```

Comment:
🛠 Read the comments below and update your code accordingly 

@vasynizami looks like your missing the initial variables needed to run the program. In order for your program to run you'll need to declare some variables

for example:

```
let yearsOfService = 7 // <- you are missing this
let netBonus // <- you are missing this
let salary = 80000 // <- you are missing this

if (yearsOfService > 5) {
  netBonus = salary * 0.05
}

console.log(netBonus)
```

Please add the variable declarations described above